Kinvey/Kinvey/CustomEndpoint.swift

```diff
@@ -36,13 +46,77 @@ open class CustomEndpoint: NSObject {
         return request
     }
     
+    //MARK: Mappable
+    
+    /// Executes a custom endpoint by name and passing the expected parameters.
+    @discardableResult
+    internal static func execute<T: Mappable>(_ name: String, params: JsonDictionary? = nil, client: Client = sharedClient, completionHandler: CompletionTHandler<T>? = nil) -> Request {
```

Comment:
These methods definitely look friendlier. I'm curious how the developer can set other request specifics such as headers or request method or `POST` body. What fields are available to custom endpoints?